AY2425S1-CS2103T-W09-3/tp
# List all internship application feature

## Description

Closes #59, #64


Update list command feature and its test case. 



## Changes made

 Add testUtils, TypicalInternshipApplications and InternshipApplicationBuilder. Removed rawtypes. Edited CommandTestUtil, and TypicalPersons



## Impact of changes
 Ensures list command works with generic model

## Testing Steps
 
1. Navigate to Main
2. Run it and type list
3. Verify that it says Listed all internship applications.


Updated tests: ListCommandTest
